stream-transformation-tool-api/src/main/java/uk/gov/justice/tools/eventsourcing/transformation/api/EventTransformation.java

```diff
@@ -17,15 +21,25 @@
      * @param event - the event to check
      * @return TRUE if the event is eligible to have the transformation applied to it.
      */
-    boolean isApplicable(final JsonEnvelope event);
+    @Deprecated
+    default boolean isApplicable(final JsonEnvelope event){
+        return false;
+    }
+
+    default TransformAction action(final JsonEnvelope event) {
```

Comment:
java doc comments. Also update the deprecated comment to point to #action()

Is actionFor a better name than action ? (just asking the question, can discuss)